src/cli/onefuzz/backend.py

```diff
@@ -181,7 +181,7 @@ def get_access_token(self) -> Any:
         if not self.config.endpoint:
             raise Exception("endpoint not configured")
 
-        if self.config.tenant_domain:
+        if "common" in self.config.authority:
```

Comment:
I think we should probably check this precisely. I think we're testing for `https://login.microsoftonline.com/common` here?

Reply:
Done.

Reply:
Done.